packages/lwc-language-server/package.json

```diff
@@ -46,7 +46,7 @@
     "vscode-html-languageservice": "^3.1.0-next.3",
     "vscode-languageserver": "^5.2.1",
     "vscode-nls": "^4.0.0",
-    "vscode-uri": "1.0.6",
+    "vscode-uri": "^2.1.2",
```

Comment:
Looks like there are more changes needed in this module since tests are failing with
```
@salesforce/lwc-language-server: src/__tests__/test-utils.ts:14:21 - error TS2339: Property 'file' does not exist on type 'typeof import("C:/Users/circleci/project/node_modules/vscode-uri/lib/umd/index")'.
@salesforce/lwc-language-server: 14     const uri = URI.file(resolve(path)).toString();
```

Reply:
Nice Catch here!
I fix this, and run all tests now: 
![image](https://user-images.githubusercontent.com/35036353/91062451-7a5a7a00-e60b-11ea-8303-2e386dd49eff.png)